ngi_pipeline/engines/sarek/process.py

```diff
@@ -44,6 +45,16 @@ def execute_process(self, command_line, working_dir=None, **extra_args):
             except RuntimeError:
                 raise
 
+    @staticmethod
+    def cleanup(target_dir):
+        """
+        Remove a specified target directory, regardless whether it's empty or not.
+
+        :param target_dir: the full path to the target directory
+        :return: None
+        """
+        shutil.rmtree(target_dir)
```

Comment:
I think a _log_ here is definitely useful if it is not logged elsewhere upstream

Reply:
:+1: